plugins/contact-resources/src/components/ContactsTabs.svelte

```diff
@@ -33,9 +33,7 @@
 </script>
 
 {#if tabs && tabs.length > 1}
-  <div class="pl-2">
```

Comment:
I would recommend just to use 'p-2' or 'p-3' class for this div.

Reply:
Used padding in Tabs